s3tests_boto3/functional/test_s3select.py

```diff
@@ -175,8 +226,17 @@ def remove_xml_tags_from_result(obj):
             continue
         result += rec + "\n" # remove by split
 
-    return result
+    result_strip= result.strip()
+    x = bool(re.search("^failure.*$", result_strip))
+    if x:
+        logging.info("invalid syntax")
```

Comment:
the original message (reside on result) contain information on point of syntax failure (the first non-accepted token)
that message should appear in log.